CHANGES.md

```diff
@@ -112,8 +115,9 @@ To be released.
     [[#909]]
  -  Fixed a bug in which pre-computed state delivery had failed when a state
     key is not an `Address` when preloading.  [[#912]]
- -  Fixed a bug where `UnexpectedlyTerminatedException` hadn't been serialized
-    with `BinaryFormatter`.  [[#913]]
+ -  Fixed a bug where `UnexpectedlyTerminatedException` and
+    `InsufficientBalanceException` hadn't been serialized with
```

Comment:
As we've never released `InsufficientBalanceException`, this doesn't have to be recorded.